index.d.ts

```diff
@@ -966,6 +966,19 @@ declare global {
     readonly hash: string
     getBytesAsync(): Promise<Uint8Array>
   }
+
+  // The plugin environment exposes the browser console API,
+  // so expected calls like console.log() still work.
+  interface Console {
+    log(message?: any, ...optionalParams: any[]): void
+    error(message?: any, ...optionalParams: any[]): void
+    assert(condition?: boolean, message?: string, ...data: any[]): void
+    info(message?: any, ...optionalParams: any[]): void
+    warn(message?: any, ...optionalParams: any[]): void
+    clear(): void
+  }
+  const console: Console
```

Comment:
mind moving this to the top with all the globals?

Reply:
Just the 'const console' bit, or the whole thing?

Reply:
just const console!

Reply:
Donezo